net-mgmt/zabbix-agent/src/opnsense/service/templates/OPNsense/ZabbixAgent/rc.conf.d

```diff
@@ -0,0 +1,6 @@
+{% if helpers.exists('OPNsense.ZabbixAgent.settings.main.enabled') and OPNsense.ZabbixAgent.settings.main.enabled|default("0") == "1" %}
+zabbix_agentd_enable=YES
+{% else %}
+zabbix_agentd_enable=NO
+{% endif %}
+zabbix_agentd_config=/usr/local/etc/zabbix_agentd.conf
```

Comment:
into enable block, setup invoke is also missing:

https://github.com/opnsense/core/blob/master/src/opnsense/service/templates/OPNsense/Proxy/rc.conf.d#L3